test/unit/lib/api-client.js

```diff
@@ -315,11 +316,60 @@ describe('API Client', () => {
                 id: '123456'
             },
             response
-        })
+        });
         return searchStudent('123456').then(data => {
             assert.deepEqual(data, response.data);
             mockServer.validate();
         });
     });
 
+    it('updateModel', () => {
```

Comment:
Nit: "it updateModel" is hardly even a phrase. It should be `describe` (with an `it` inside) not `it`.